### DISCUSSION TITLE
How to auto-size q-table for full page height without page scrolling

### DISCUSSION THREAD

**Discussion by akrtkv** [quasarframework/quasar#8773] [General - Components / Directives / etc] [answered]
![123](https://user-images.githubusercontent.com/48287188/113557633-ffc4f480-9606-11eb-8786-755036bafd4e.jpeg)
So, question in the title. I can use (.my-sticky-virtscroll-table height: 91vh) but if screen size less than full hd the page show scrollbar.

**Comment by metalsadman** [comment, marked as answer]
@deepfunction checkout https://quasar.dev/vue-components/resize-observer#introduction, then set your qtable's height dynamically base off the `resize` event data.
